movie/src/components/SortCard/SortCard.jsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Collapse from "@mui/material/Collapse";
+import Typography from "@mui/material/Typography";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import "./SortCard.css";
+import { useState } from "react";
```

Comment:
you're import from `react` twice. please remove the first line. Since you're on react-18,
So `import * as React from "react"` is no longer needed